Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/AppBar/AppBar.cs

```diff
@@ -44,135 +44,181 @@ public enum AppBarStateEnum
 
         #endregion
 
-        #region Public properties
+        #region Private Serialized Fields with Public Properties
 
-        /// <summary>
-        /// Where to display the app bar on the y axis
-        /// This can be set to negative values
-        /// to force the app bar to appear below the object
-        /// </summary>
-        public float HoverOffsetYScale
+        [Header("Target Bounding Box")]
+        [SerializeField]
+        private BoundingBox boundingBox = null;
+
+        public BoundingBox TargetBoundingBox
```

Comment:
could you add /// comments and tooltips to all those properties please? I know this was already missing before but it would be great to get this file up to our coding/doc guideline/standards when you're already changing those properties

Reply:
Can do

Reply:
Done

Reply:
Thanks for doing this!